### PR TITLE
Removed the dispatch and set max value for slider

### PR DESCRIPTION
**Pull request by mAkeddar**
I forgot a dispatch in enterRoute, causing the first issue
Second one was due to default value of slider's max = 100, changed it to 25

### CODE REVIEW

File: src/Pages/Driver/enterRoute.tsx [hunk 1]
```diff
@@ -19,9 +18,9 @@ const MARKS = {
   0: '0 km',
```
Comment (mischakolbe) [inline src/Pages/Driver/enterRoute.tsx:18]: I wonder... Does it make sense to even have a "0 km" detour option? I don't think this sends the correct message to drivers; they'll always have to make a bit of a detour, except for maybe 0.5% of the cases.
What do you think, @conradbielitz , from a business perspective?

**Review by mischakolbe — CHANGES_REQUESTED** [review]
Just that question about the 0km detour. Apart from that it looks good.

File: src/Pages/Driver/enterRoute.tsx (hunk 1 above)
Comment (mAkeddar) [reply src/Pages/Driver/enterRoute.tsx:18]: Makes sense for me too!

**Comment by mAkeddar** [comment]
Start it at 5

**Pull request merged**